src/ProjectOrigin.WalletSystem.IntegrationTests/Repositories/CertificateRepositoryTests.cs

```diff
@@ -173,6 +174,71 @@ public async Task GetAllOwnedCertificates()
         );
     }
 
+    [Fact]
+    public async Task GetAllOwnedCertificates_WalletAttributes()
+    {
+        // Arrange
+        var endpointPosition = 1;
+        var registry = _fixture.Create<string>();
+        var owner = _fixture.Create<string>();
+        var wallet = await CreateWallet(owner);
+        var endpoint = await CreateWalletEndpoint(wallet);
+
+        var certificateId = Guid.NewGuid();
+        WalletAttribute walletAttribute = new WalletAttribute
```

Comment:
suggest renaming this to `assetIdWalletAttribute`

Reply:
done